src/ORM/Connect/DBSchemaManager.php

```diff
@@ -60,10 +61,11 @@ abstract class DBSchemaManager
      * @param string $table
      * @param string $class
      *
-     * @deprecated 4.0.0:5.0.0
+     * @deprecated 4.0.0
      */
     public static function showTableNameWarning($table, $class)
     {
+        Deprecation::notice('4.12.0');
```

Comment:
```suggestion
        Deprecation::notice('4.0.0');
```
Add a message to both this notice and the phpdoc

Reply:
Updated